src/status_im/ui/screens/browser/db.cljs.orig

```diff
@@ -0,0 +1,41 @@
+(ns status-im.ui.screens.browser.db
```

Comment:
looks like a failed merge

Reply:
*.orig file committed by mistake. Should be removed.